src/main/java/pl/project13/maven/git/GitDescribeConfig.java

```diff
@@ -28,7 +28,7 @@
    * If you don't use describe, you can always disable it and make the build a bit faster.
    *
    * Although it's highly recommended to use <pre>git-describe</pre> to identify your build state,
-   * so think twice before disabeling it.
+   * so think twice before disabling it.
    *
    * @parameter default-value=false
```

Comment:
hm hm, does this not need to be moved to @Parameter too? It's a nested config object.